wdl-engine/src/backend/docker.rs

```diff
@@ -487,6 +487,7 @@ impl TaskExecutionBackend for DockerBackend {
         /// Amount of memory to reserve for the cleanup task.
         const CLEANUP_MEMORY: f64 = 0.05;
 
+        let work_dir = work_dir.as_local().expect("path should be local");
```

Comment:
Would it be worth a safety comment just stating that this is enforced by the Docker backend?